tests/ui/errors/remap-path-prefix-sysroot.rs

```diff
@@ -0,0 +1,12 @@
+// run-fail
+// check-run-results
+
+// exec-env:RUST_BACKTRACE=full
```

Comment:
I don't think we can make CI and the bors tests pass with a test that panics and with `RUST_BACKTRACE` set. So I think we have to remove this test.